app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardViewModel.kt

```diff
@@ -29,21 +29,11 @@ class DashboardViewModel(
         const val DEFAULT_YELLOW_WARNING_QUARANTINE = 168 // hours
     }
 
-    val showMicrophoneExplanationDialog: Boolean
-        get() = dashboardRepository.showMicrophoneExplanationDialog
-
-    // TODO: 27/05/2020 dusanjencik: Decide if we want to enable it automatically or not
-//    var wasServiceEnabledAutomaticallyOnFirstStart: Boolean
-//        get() = coronaDetectionRepository.serviceEnabledOnFirstStart
-//        set(value) {
-//            coronaDetectionRepository.serviceEnabledOnFirstStart = value
-//        }
-
-    var batteryOptimizationDialogShown = false
-
-    fun observeSavedEncounters(): Observable<Int> {
-        return dashboardRepository.observeSavedEncountersNumber()
-    }
+    var wasServiceEnabledAutomaticallyOnFirstStart: Boolean
```

Comment:
there is no Service anymore

Reply:
Done in https://github.com/austrianredcross/stopp-corona-android/pull/83#discussion_r433731286